crates/uv-python/src/discovery.rs

```diff
@@ -313,7 +314,10 @@ fn python_executables_from_installed<'a>(
     version: &'a VersionRequest,
     implementation: Option<&'a ImplementationName>,
     preference: PythonPreference,
+    preview: PreviewMode,
```

Comment:
Do we need preview mode to be threaded through here? Don't we always want to discover minor version links if they exist, regardless of whether preview is enabled?

Reply:
Even when preview is disabled, `PythonMinorVersionLink` will check if the symlink exists. We could pass `PreviewMode::Disabled` directly into `PythonMinorVersionLink::from_installation` here, but I think that's too prone to error. My goal was to centralize the logic around how to handle preview states.